src/main/java/com/bootcamp/demo/Controller/CardController.java

```diff
@@ -0,0 +1,21 @@
+package com.bootcamp.demo.Controller;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.cloud.FirestoreClient;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/card")
```

Comment:
Very good annotations choice. 
You could also add the `produces = APPLICATION_JSON_VALUE` (org.springframework.http.MediaType.APPLICATION_JSON_VALUE) param for `@RequestMapping` (`consumes` param should be added only for post mappings)

Reply:
I've added the produces argument now